cf/tasks/package-bits.sh

```diff
@@ -30,8 +30,9 @@ do
     mkdir tmp
 
     # kube configs
-    # helm charts
     unzip s3.kube-dist/scf-kube-*.zip -d tmp
+    # helm charts
+    unzip s3.helm-dist/scf-helm-*.zip -d tmp
```

Comment:
Hm. IIRC helm has a `helm` sub dir, and the other two `cf` and `uaa`. Should have no conflict.
Regardless, I will check what the concourse gives me.